src/main/java/com/internet/shop/model/User.java

```diff
@@ -23,6 +24,21 @@ public User(Long id, String name, String login, String password, Set<Role> roles
         this.roles = roles;
     }
 
+    public User(String name, String login, String password, byte[] salt) {
+        this.name = name;
+        this.login = login;
+        this.password = password;
+        this.salt = salt;
+    }
+
+    public User(Long id, String name, String login, String password, byte[] salt) {
```

Comment:
I am conserned about so many constructors in this class) this is not a good practice and should better be refactored

Reply:
agree, they were unused, already deleted them